internal/command/jobrunnergroup/command.go

```diff
@@ -0,0 +1,26 @@
+package jobrunnergroup
+
+import (
+	"github.com/signadot/cli/internal/config"
+	"github.com/spf13/cobra"
+)
+
+func New(api *config.API) *cobra.Command {
+	cfg := &config.JobRunnerGroup{API: api}
+
+	cmd := &cobra.Command{
+		Use:     "jobrunnergroup",
+		Short:   "Inspect and manipulate jobrunnergroup",
+		Aliases: []string{"jrg"},
+	}
+
+	// Subcommands
+	cmd.AddCommand(
+		newGet(cfg),
+		newList(cfg),
+		newApply(cfg),
+		newDelete(cfg),
```

Comment:
@foxish : I realized while testing that deleting a JRG deletes all the corresponding jobs as well. I wonder if we should require the user to pass a `--force` flag, or respond to a prompt so that the user is informed about this destructive activity.

Reply:
Not MVP scope, because there may be an actual `--force` option we add to not have the JRG drain gracefully.

Reply:
ack